code/modules/clothing/suits/cloaks.dm

```diff
@@ -50,12 +50,28 @@
 	name = "captain's cloak"
 	desc = "Worn by the commander of Space Station 13."
 	icon_state = "capcloak"
-	
+
 /obj/item/clothing/neck/cloak/hop
 	name = "head of personnel's cloak"
 	desc = "Worn by the Head of Personnel. It smells faintly of bureaucracy."
 	icon_state = "hopcloak"
 
+/obj/item/clothing/suit/hooded/cloak/goliath
+	name = "goliath cloak"
+	icon_state = "goliath_cloak"
+	desc = "A staunch, practical cape made out of numerous monster materials, it is coveted amongst exiles & hermits."
+	allowed = list(/obj/item/device/flashlight, /obj/item/weapon/tank/internals, /obj/item/weapon/pickaxe, /obj/item/weapon/twohanded/spear, /obj/item/weapon/twohanded/bonespear, /obj/item/organ/hivelord_core/legion, /obj/item/weapon/kitchen/knife/combat/bone, /obj/item/weapon/kitchen/knife/combat/survival)
+	armor = list(melee = 35, bullet = 10, laser = 25, energy = 10, bomb = 25, bio = 0, rad = 0, fire = 60, acid = 60) //a fair alternative to bone armor, requiring alternative materials and gaining a suit slot
+	hoodtype = /obj/item/clothing/head/hooded/cloakhood/goliath
+	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS
```

Comment:
Thanks for picking up on that, it completely flew by me.

Legs is _required sorta_ because its a suit replacement for bone armor (which has legs), unless of-course i make some sort of bone greaves item later and remove the legs from bone armor to make it fair. (its kind of iffy because the dragon suit also pulls over the feet, and imminently adding extra armor that isn't totally rubbish, or hard to obtain will mean the dragon suit gets a buff indirectly)

Ill make the changes as requested and mill over the balancing later, i have a plan for a item that is not bone greave shoes but still aesthetically appropriate I would think.